docs/develop/support-ie-11.md

```diff
@@ -48,7 +50,34 @@ Importing the entire core-js library will import all core-js features. You can a
 
 For a sample add-in that uses core.js, see [Word Add-in Angular2 StyleChecker](https://github.com/OfficeDev/Word-Add-in-Angular2-StyleChecker).
 
-## Testing an add-in on Internet Explorer
+## Determine at runtime if the add-in is running in Internet Explorer
```

Comment:
i think we are missing to mention how to setup office so it uses trident. would be awesome to include it here